docker-compose.yml

```
@@ -6,7 +6,7 @@ db:
   environment:
     - MYSQL_ROOT_PASSWORD=my-super-secret-password
 web:
-  image: openxpki:latest
+  build: ./
```

This should be for dev/test environments only, right?

---

Oh yes, my mistake sorry.
```suggestion
  + image: openxpki:latest
  - build: ./
```

---

No worries, looks good.